src/web/nootils.js

```diff
@@ -31,16 +31,17 @@ module.exports = function(settings) {
     getSchedule: function(name) {
       return _.findWhere(settings.tasks.schedules, { name: name });
     },
-    getMostRecentTimestamp: function(reports, form) {
-      var report = this.getMostRecentReport(reports, form);
+    getMostRecentTimestamp: function(reports, form, fields=null) {
+      var report = this.getMostRecentReport(reports, form, fields);
       return report && report.reported_date;
     },
-    getMostRecentReport: function(reports, form) {
+    getMostRecentReport: function(reports, form, fields=null) {
```

Comment:
a default value of null doesn't seem to have an impact. might as well remain undefined. is it useful?

Reply:
It's not. Fixing it.